lib/connect.js

```diff
@@ -278,6 +279,12 @@ function connect(token, options) {
  * @property {boolean|CreateLocalTrackOptions} [audio=true] - Whether or not to
  *   get local audio with <code>getUserMedia</code> when <code>tracks</code>
  *   are not provided.
+ * @property {boolean} [automaticSubscription=true] - By default, you will subscribe
+ *   to all RemoteTracks shared by other Participants in a Room. You can now override this
```

Comment:
@makarandp0 

> You can now override this behavior by setting this flag to false. This will make sure that you will not subscribe to any RemoteTrack in a Group or Small Group Room. ...